docs/doc/04-security/00-access-control/10-access-control-privileges.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Better to move here?

![image](https://user-images.githubusercontent.com/172204/223927774-8dd8700d-fab5-4f75-9c14-de4286614f64.png)

Reply:
Before Use cases after SQL Functions